frontend/src/js/KonvaManager/PencilManager.ts

```diff
@@ -38,11 +38,15 @@ export default class PencilManager {
     const newLine = new PencilLine(config, layer);
     this.isDrawing = true;
     this.currentDrawing = newLine;
-    this.currentDrawing.layer.add(this.currentDrawing);
-    this.currentDrawing.redraw();
+    this.draw(this.currentDrawing);
```

Comment:
This probably shouldnt be drawn here right away , but it doesnt matter now until it s not sent through hub